executor/modes/local/runner.py

```
@@ -344,156 +349,140 @@ async def _on_client_created(self, task_id: int) -> None:
+    def _make_emitter_report_progress(self, emitter: ResponsesAPIEmitter) -> callable:
+        """Create a progress callback that reports via emitter.
+
+        Args:
+            emitter: ResponsesAPIEmitter instance for sending events
+
+        Returns:
+            A callback function for reporting progress
+        """
 
         def report(
             progress: int,
```

Ruff (0.15.0)

[warning] Unused function argument: `progress`

(ARG001)